docusaurus/docs/introduction/backend.md

```diff
@@ -0,0 +1,92 @@
+---
+id: backend-plugins
+title: Backend plugins
+---
+
+# Backend plugins
+
+The Grafana plugin system for backend development allows you to integrate Grafana with virtually anything and offer custom visualizations. This document explains the system's background, use cases, and key features.
+
+## Background
+
+Grafana added support for _frontend plugins_ in version 3.0 so that the Grafana community could create custom panels and data sources. It was wildly successful and has made Grafana much more useful for our user community.
+
+However, one limitation of these plugins is that they run on the client side, in the browser. Therefore, they can't support use cases that require server-side features.
+
+Since Grafana v7.0, we have supported server-side plugins that remove this limitation. We use the term _backend plugin_ to denote that a plugin has a backend component. A backend plugin usually requires frontend components as well. For example, some backend data source plugins need query editor components on the frontend.
+
+## Use cases for implementing a backend plugin
+
+The following examples give some common use cases for backend plugins:
+
+- Enable [Grafana Alerting](https://grafana.com/docs/grafana/latest/alerting/) for data sources.
+- Connect to SQL database servers and other non-HTTP services that normally can't be connected to from a browser.
+- Keep state between users, for example, by query caching for data sources.
+- Use custom authentication methods and/or authorization checks that aren't supported in Grafana.
+- Use a custom data source request proxy (refer to [Resources](#resources) for more information).
+
+## Grafana backend plugin system
+
+The Grafana backend plugin system is based on HashiCorp's [Go Plugin System over RPC](https://github.com/hashicorp/go-plugin). Our implementation of the Grafana server launches each backend plugin as a subprocess and communicates with it over [gRPC](https://grpc.io/).
+
+### Benefits for plugin development
+
+Grafana's approach has benefits for developers:
+
+- **Stability:** Plugins can't crash your Grafana process: a panic in a plugin doesn't panic the server.
+- **Ease of development:** Plugins can be written in any language that supports gRPC (for example, write a Go application and run `go build`).
```

Comment:
@jackw @josmperez This statement feels very close to https://github.com/grafana/plugin-tools/pull/296#discussion_r1257897345

How about we remove and for now have a generic statement like:

Ease of development: Grafana provides an officially supported SDK for Go and Tooling to help create plugins.

Reply:
done